packages/common/src/core/HubItemEntity.ts

```diff
@@ -330,4 +343,31 @@ export abstract class HubItemEntity<T extends IHubItemEntity>
     }
   }
   //#endregion IWithFeaturedImageBehavior
+
+  //#region IWithVersioningBehavior
```

Comment:
This is perfect - keep the class methods as lean as possible... one thing I'd suggest is passing `requestOptions` or one of it's sub-types vs the whole context object. That's more the hub.js pattern for functions